### ISSUE TITLE
[feature request] auto text-overflow to comments

### ISSUE THREAD

**Issue by ismirth** [tuskyapp/Tusky#1425]
when copying text or posts are too long... I seen a feature on a different app but I don't remember which one... It automatically with cutaway the extra text and add [cont...] Where the character number runs out... And then add consecutive on to the post of all the rest of the text so if you wanted to post something with ten thousand char long, you could do that, comments... Numbered and in the correct order. I'm currently having to copy and paste by hand on my phone and that's very tedious. An auto overflow feature would be really fantastic!

* * * *
- Tusky Version: 8.1
- Android Version: 9
- Android Device:
- Mastodon instance (if applicable):

- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by charlag** [comment]
Hi!
Do I understand it correctly that you would like it so: when you copy the long text it's not copied completely?
If not, please, could you explain it clearly? Which comments do you mean?

**Comment by ismirth** [comment]
No, what I am saying is thatwhen I create a post where I am copying text
that is beyond the character limit, that it should automatically truncate
it and copy the excess into to subsequent comments.

**Comment by charlag** [comment]
ah, now I understand, thanks
When you paste long text into the editor, you want it to be separated into parts and posted as replies.
This would be a nice feature but maybe we should allow writing the whole thread by hand too? It's not an easy task but maybe worth looking into in the future.

**Comment by ismirth** [comment]
however the text is entered into the box... If it's more characters than is
allowed the overflow should be automatically added into the comments...
What do you copy and paste that are with addicted by hand.

previously I was using a difference app to post to Mastodon and they had
this function... but then I uninstalled it and now I can't get it back so I
started using your guys's app which holistically I like much better... But
I am greatly saddened that it is lacking this feature.

I hope that you guys can make it happen! Thank you so much.

**Comment by kefir-** [comment]
I second this. Not so bothered with the copy/paste scenario. I'd like to see functionality to add one or more replies, chained after one another as a thread, and allowing them all to be edited until they are all posted. 

Not so sure about the auto-split on text overflow, that might get annoying actually. The author will probably want to plan where the split goes, and whether they want to add the typical "1/3" thread length indicator.

**Comment by maninthemiddle01** [comment]
This is very useful indeed and is already implemented in [Fedilab](https://fedilab.app/).
